.gitignore

```diff
@@ -1,5 +1,6 @@
 *.avm
 *.nef
+*~
```

Comment:
wow. what is it used for?

Reply:
File backup. I thought every .gitignore has this pattern.